flink-runtime-web/web-dashboard/src/app/pages/job/overview/backpressure/job-overview-drawer-backpressure.component.html

```diff
@@ -17,14 +17,17 @@
   -->
 
 <nz-table
-  class="no-border small full-height"
-  [nzSize]="'small'"
+  nzSize="small"
   [nzLoading]="isLoading"
   [nzData]="listOfSubTaskBackpressure"
   [nzScroll]="{ y: 'calc( 100vh - 72px )' }"
```

Comment:
`100vh - 72px` may cause scrollbar displaying, maybe we can change it to be `100% - 72px`?

Reply:
This is a bug brought by FLINK-27441. I will fix it here.